README.md

```diff
@@ -0,0 +1,186 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>School Library</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 School Library <a name="about-project"></a>
+
+**School Library** is an open-source platform storing students, teachers and bowroing books.You can easily manage getting and receving of books in school library.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS/SASS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Render</li>
+    <li>Ruby</li>
+   
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[creating students]**
+- **[creating teachers]**
+- **[managing school library]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+(under work)
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- You'll need any code editor 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com:https://github.com/ghreza-crypto/school-library.git
+  cd school-library 
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
```

Comment:
[OPTIONAL] Kindly note that your `usage`, `install`, and `run test` section reflects that of a react or javascript-related project🥲 Since this project is not managed by node, it will be more professional to give the right commands for these sections👍.